prime-router/docs/universal-pipeline/convert.md

```diff
@@ -1,20 +1,27 @@
+  3. Generate a new Report
+  4. Schedule the Route step for the new Report
 
 ## Conversion
 The first and most significant aspect of the convert step is the conversion of a message into the FHIR format. The process of this conversion depends on the incoming message's format:
 
-- If the incoming message is HL7v2, we use the [LinuxForHealth HL7 to FHIR Converter library](https://github.com/LinuxForHealth/hl7v2-fhir-converter#linuxforhealth-hl7-to-fhir-converter) with our custom configurations from the [fhir_mapping folder](https://github.com/CDCgov/prime-reportstream/tree/master/prime-router/metadata/fhir_mapping/hl7) to handle the conversion. See the [HL7v2-to-FHIR configurations](#HL7v2-to-FHIR-configurations) section for more details.
-- If the incoming message is FHIR, then there's nothing to be done, so we can continue onto sender transforms.
-- Currently, no other input formats are supported in the Universal Pipeline, but to support new input types, they would need to be translated to FHIR by some other method.
+- If the incoming message is HL7v2, ReportStream uses the [LinuxForHealth HL7 to FHIR Converter library](https://github.com/LinuxForHealth/hl7v2-fhir-converter#linuxforhealth-hl7-to-fhir-converter) with custom configurations from the [fhir_mapping folder](https://github.com/CDCgov/prime-reportstream/tree/master/prime-router/metadata/fhir_mapping/hl7) to handle the conversion. See the [HL7v2-to-FHIR configurations](#HL7v2-to-FHIR-configurations) section for more details.
+- If the incoming message is FHIR, then there's nothing to be done
+- Apply Sender transforms specified by Sender configuration 
+
+> See [HL7-to-FHIR Translations](../design/design/translations.md#hl7-to-fhir-translations) for more information regarding configuration of HL7 to FHIR translations.
 
-### HL7v2-to-FHIR configurations
-[section all about the mapping configurations in `fhir_mapping`, link to separate page if necessary]
+> Currently, no other input formats besides HL7v2 and FHIR are supported in the Universal Pipeline. In order to support new formats, they would need to be translated to FHIR by some new method.
 
 ## Sender Transforms
-After the conversion to FHIR format is complete, we can now apply sender transforms. We use these transforms to make sender-specific adjustments to the resulting FHIR bundle, so we can think of this as a FHIR-to-FHIR transform. The class `FhirTransformer` is used to perform these kinds of transforms.
+After the conversion of an Item to the FHIR format, an additional transform can run on the resulting FHIR bundle. This transform is called a "sender transform" because it is used to make sender-specific adjustments to the FHIR bundle. These transforms can also run in the [translate](translate.md) step, in the case a receiver wants to receive FHIR, where they would be called "receiver transforms" so the preferred term for these transforms is "FHIR to FHIR transforms."
```

Comment:
Not sure exactly how to rephrase this, but I don't know if we need to say the preferred term overall is "FHIR to FHIR transforms", since it is useful sometimes to refer to either sender or receiver transforms specifically. We might not need to refer to receiver transforms at all here, since it's covered in `translations.md`